src/bp-templates/bp-nouveau/includes/activity/template-tags.php

```diff
@@ -478,6 +478,38 @@ function bp_nouveau_get_activity_entry_buttons( $args ) {
 				),
 			);
 
+			if ( bp_is_activity_edit_enabled( false ) ) {
+
+				$activity_edit_time = (int) bp_get_activity_edit_time(); //for 10 minutes, 600
+
+				//bp_dd_get_time() is not exists here.
+				$bp_dd_get_time            = current_time( 'timestamp' );
```

Comment:
@shohel why we are using this `current_time( 'timestamp' )` instead of the `bp_core_current_time()` builin function? Any reason for this?

Reply:
Changed current_time with bp_core_current_time()

Reply:
Changed current_time with bp_core_current_time()